persistence-jdbc/core/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/JndiConfigurator.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.jdbc
+
+import javax.naming.InitialContext
+
+import com.typesafe.config.Config
+import play.api.db.DBApi
+
+import scala.util.Try
+
+private[lagom] trait JndiConfigurator
+
+private[lagom] object JndiConfigurator {
+
+  def apply(dbApi: DBApi, config: Config): JndiConfigurator = new JndiConfiguratorImpl(dbApi, config)
+
+  private final class JndiConfiguratorImpl(dbApi: DBApi, config: Config) extends JndiConfigurator {
```

Comment:
The trait and class seem superfluous... this is really just a procedure that needs to run, isn't it? Could be a unit function?

Reply:
Sure.  I will adapt it.